doc/man/salt.7

```diff
@@ -353551,15 +353551,16 @@ Here is an example of such a configuration, which should speak by itself:
 .ft C
 ext_pillar:
   \- makostack:
```

Comment:
The man pages should not be edited directory. These are auto generated at release time.

Reply:
I rolled back these changes.